### PR TITLE
passwordReset component

### PR DESCRIPTION
**Pull request by tmansuroglu**
This is separated from #81

### CODE REVIEW

**Review by ssinikoglu — APPROVED** [review]
Comments are well-thought and decent, code looks nice in general.

**Review by louisrli — COMMENTED** [review]
ıt looks fıne but tests need to pass

**Comment by tmansuroglu** [comment]
test needs file from register page pr

**Comment by tmansuroglu** [comment]
@louisrli can re review this?

**Review by louisrli — COMMENTED** [review]
proofread this pr please

File: src/components/userauth/ResetPasword/index.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+import ErrorMessage from "../ErrorMessage";
+
+const ResetPassword = ({
+  // // check containers/LoginRegister/index.jsx to see details of functions
+  onSubmit,
+  handleLogin,
+  // string
+  error,
+  // string
+  message,
```
Comment (louisrli) [inline src/components/userauth/ResetPasword/index.jsx]: what message

**Review by louisrli — APPROVED** [review]

File: src/components/userauth/ResetPasword/index.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.loginContainer__alert {
```
Comment (louisrli) [inline src/components/userauth/ResetPasword/index.css]: there is a typo in ResetPasword in the directory

**Review by louisrli — CHANGES_REQUESTED** [review]
fix files

File: src/components/userauth/ResetPasword/index.css (hunk 2 above)
Comment (tmansuroglu) [reply src/components/userauth/ResetPasword/index.css]: PTAL

File: src/components/userauth/ResetPasword/index.jsx (hunk 1 above)
Comment (tmansuroglu) [reply src/components/userauth/ResetPasword/index.jsx]: comment added PTAL

File: src/components/userauth/ResetPassword/index.jsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
+import { Alert, Input } from "antd";
+import { useTranslation } from "react-i18next";
+import React, { useState } from "react";
+import { validateEmail, createErrorClassName } from "../registerHelper";
+import ErrorMessage from "../ErrorMessage";
+
+const ResetPassword = ({
+  // // check containers/LoginRegister/index.jsx to see details of functions
```
Comment (louisrli) [inline src/components/userauth/ResetPassword/index.jsx]: double comment

**Review by louisrli — APPROVED** [review]

**Comment by YamanSammani** [comment]
@Ammar-64 , Kindly merge with dev.

**Pull request merged**